src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,34 @@
-export const PostInfo = () => <>Put the post here</>;
+/* eslint-disable */
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  let isNoComments = false;
+
+  if (post.comments.length == 0) {
```

Comment:
Consider using the strict equality operator `===` instead of `==` for the comparison `post.comments.length == 0`. This ensures type safety and is a best practice in JavaScript.